css/style.css

```diff
@@ -449,6 +449,95 @@ h3.lang-1 {
   right: 80px;
 }
 
+.contact-form {
+  text-align: center;
+  border-top-left-radius: 80px 80px;
+  padding: 103px 24px 45px;
+  background: #6070ff no-repeat right/60%  url(../img/contact-form-shapes/contact\ form\ shapes-mobile@2x.png);
+  flex-shrink: 0;
+}
+
+.contact-form h2 {
+  font-family: Poppins, sans-serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  color: #fff;
+  padding-bottom: 12px;
+}
+
+.contact-form .contact-paragraph {
+  text-align: center;
+  font-family: Poppins-Thin, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  color: #ebebff;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-bottom: 52px;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.input {
+  border: 0;
+  padding: 5%;
+  border-radius: 12px;
+  font-family: Poppins, sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  color: #172b4d;
+}
+
+.input-text {
+  border: 1px solid #cfd8dc;
+  padding: 5%;
+  height: 170px;
+  flex-shrink: 0;
+  border-radius: 8px;
+  font-family: Poppins, sans-serif;
+  color: #b3bac5;
+  padding-bottom: 22px;
+  width: 100%;
+}
+
+.button-contact-form {
+  display: inline-flex;
+  align-items: center;
+  margin-top: 12px 16px;
+  margin-right: 209px;
+  padding: 12px 16px;
+  width: 142px;
+  height: 48px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background: #f7f7f9;
+  color: #6070ff;
+  font-size: 17px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.51px;
+  padding-bottom: 45px;
+  cursor: pointer;
+}
+
+.button-contact-form:hover {
+  background-color: salmon;
+  color: white;
+  border: 1px solid salmon;
+}
+
```

Comment:
- [`OPTIONAL`] Please ensure that the colors used in your `hover` effect align with the corresponding colors specified in the `Figma` design. It's crucial to replicate the `Figma` design as closely as possible, as stated in the project requirements. This will help maintain consistency and ensure that the final design matches the intended visual style. Please review and update the colors used in your `hover` effect to match the design specifications provided in the `Figma` design. See below for the color details:
<img width="900" alt="Screenshot 2023-07-11 at 19 18 52" src="https://github.com/IvonneBenitesRodriguez/microverse_mobile_portfolio/assets/107192076/1e7f8678-ea45-4c6e-b2c6-e315ad5029a4">